spec/Ix.ts

```diff
@@ -1,7 +1,9 @@
 /* tslint:disable */
-
 (<any>global).window = (<any>global).window || global;
 
+// Require rxjs first so we pick up its polyfilled Symbol.observable
```

Comment:
Just for the record, this line can go away once we upgrade to Rx 6, correct?

Reply:
Yes